tutorials/cp-aibus-dox-ui-gen-ai/cp-aibus-dox-ui-gen-ai.md

```diff
@@ -0,0 +1,218 @@
+---
+parser: v2
+auto_validation: true
+time: 15
+tags: [tutorial>beginner, topic>machine-learning, topic>artificial-intelligence, topic>cloud, software-product>sap-business-technology-platform, software-product>sap-ai-business-services, software-product>document-information-extraction]
+primary_tag: topic>machine-learning
+---
```

Comment:
Please add author and author profile to meta data.

Reply:
@jmmargo: Done!